### DISCUSSION TITLE
Alert Manager in Production - Prometheus/Mimir/Loki or Embedded Grafana?

### DISCUSSION THREAD

**Discussion by pmanno** [grafana/grafana#50540] [Alerting]
Hi all,

Running Grafana OSS v8.5

Wondering if it's recommended to use the embedded alert manager for production alerting, or if it's better to use the Mimir/Prometheus alert managers?

Thanks,
Paul M.

**Comment by gotjosh** [comment]
Hi @pmanno, I believe this is a duplicate of: https://github.com/grafana/grafana/discussions/45773#discussioncomment-2235421

The tl;dr is that we wouldn't recommend using the Grafana Alertmanager as a centralised piece.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
